src/pages/Authors/Authors.js

```diff
@@ -1,7 +1,63 @@
+import {useEffect, useState} from "react";
+import {ListItem, Pagination} from "@mui/material";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import List from "@mui/material/List";
+import Divider from "@mui/material/Divider";
+import Grid from "@mui/material/Grid";
+import api from "api/api";
+import {AUTHORS_PER_PAGE} from "constants/constants";
+import Author from "components/Author";
 import classes from "./Authors.module.scss";
 
+
+
 const Authors = () => {
-    return <h1>Authors page!</h1>
+    const [authors, setAuthors] = useState([]);
+    const [page, setPage] = useState(1);
+    const [totalPages, setTotalPages] = useState(1);
+
+    useEffect(() => {
+        api.getAuthors(page).then(res => {
+            setAuthors(res.authors);
+            setTotalPages(Math.ceil(res.total / AUTHORS_PER_PAGE));
+
+        });
+    }, []);
+
+    useEffect(() => {
+        api.getAuthors(page).then(res => {
+            setAuthors(res.authors);
+            setTotalPages(Math.ceil(res.total / AUTHORS_PER_PAGE));
+        });
+    },[page]);
```

Comment:
do you need both of these? If you `console.log` inside both `useEffect` do they both execute on the first render?